examples/basic/axes/twin_axes.py

```diff
@@ -11,23 +11,26 @@
 
 from bokeh.models import LinearAxis, Range1d
 from bokeh.plotting import figure, show
+from bokeh.palettes import Sunset6
```

Comment:
missed this ealier, imports are out of order (triggering an isort check failure)
```suggestion
from bokeh.plotting import figure, show
from bokeh.palettes import Sunset6
```